packages/app/src/SessionSubpanel.tsx

```diff
@@ -379,7 +379,7 @@ export default function SessionSubpanel({
       end,
       whereLanguage,
       searchedQuery,
-      filteredEventsFilter,
+      allEventsFilter,
```

Comment:
@ernestii just curious here...what is `filteredEventsFilter` vs `allEventsFilter` and why does this work? Can you add a few comments for future devs?

Reply:
`filteredEventsFilter` is used in the `Highlighted` tab – good point, updated the var name and added comment

![Screenshot 2025-04-29 at 7 47 30 PM](https://github.com/user-attachments/assets/3e7be0b6-725e-41ec-8c7b-72291e27ae7d)